src/stores/auth.js

```diff
@@ -160,6 +162,27 @@ export const useAuthStore = defineStore('auth', () => {
         }
     }
 
+    const getUserList = async (page, limit) => {
+        if (!state.userToken) {
+            console.error("Authentication token not found.");
+            return;
+        }
+        try {
+            const response = await axios.get(`${apiUrl}/user/list/${page}?limit=${limit}`, {
+                headers: {
```

Comment:
you should create axios interceptor to manage headers. it should not be in code like this @amrit56789